js/app.js

```diff
@@ -120,11 +120,10 @@ function preload() {
 }
 
 function runScript(){
-    var globalScope = {};
```

Comment:
No more scope object to pass around. Blocks elements are their own scopes, using the DOM structure for scope control. For runtime functions it means `this` now refers to the DOM block we're calling `run` on, rather than a plain JavaScript object.